source/chapters/configuration.rst

```diff
@@ -1,7 +1,31 @@
+.. include:: /shortcuts.rstext
+
 .. _configuring-mixxx:
 
-Configuring Mixxx
-*****************
+Mixxx Setup
+***********
+
+.. _djing-open:
+
+Opening Mixxx
+=============
+|logo| Once you've :ref:`installed Mixxx <installing-mixxx>`, start by opening
+Mixxx and import your music to the Mixxx library.
+
+**Windows**
+  Double-click the Mixxx icon on the Desktop. Alternatively, browse your Windows
+  start menu and click the Mixxx icon, or perform a search for 
+  :file:`Mixxx.exe`.
+
+**Mac OSX**
+  Double-click the Mixxx icon in the :file:`Applications` folder. Alternatively,
+  drag the Mixxx icon to the dock and double-click it there, or perform a search
```

Comment:
"or perform a search in Spotlight for "Mixxx"

Most Mac users won't understand that Mixxx has a .app suffix